### PR TITLE
WIP: 3.1 update (#15)

### LINKED ISSUE #15: Update to 3.1

### PR DESCRIPTION
**Pull request by Christian-TD**
This updates aoa cli to 3.1 and adds ldap support to scheduler.

closes #15

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ trigger:
       - '*'
 
 variables:
+  ## TODO: create azure connection to AWS ECR
```
Comment (willfleury) [inline azure-pipelines.yml:15]: remove this comment and create a ticket if you need to

**Review by willfleury — APPROVED** [review]
looks good - just remove the TODO comment

File: azure-pipelines.yml (hunk 1 above)
Comment (Christian-TD) [reply azure-pipelines.yml]: done

**Comment by Christian-TD** [comment]
closed by https://github.com/ThinkBigAnalytics/AoaExternalDockerImages/pull/21

**Pull request closed without merging**